### PR TITLE
Refactor/quote block

### PR DESCRIPTION
**Pull request by vvandijck**
This pull-request includes:
- Remamed `text-block` to `colored-block` to better reflect it's purpose.
- `colored-block` is core styling to create a colored box using a color coming from DatoCMS.

### CODE REVIEW

File: components/image-combo/image-combo.less [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		margin-bottom: 0;
 	}
 
-	.text-block {
+	.colored-block {
```
Comment (Siilwyn) [inline components/image-combo/image-combo.less]: This class is not used anywhere, `<ImageCombo>` is used in the team overview component with no props and in the case page with the prop `classes={hasTextCard ? 'image-combo-text' : ''}`.

So the class can be deleted. And if you have the time I would remove the `classes` prop and replace it with something like `hasText`? Having a generic `classes` prop seems like a bad idea.

Comment (vvandijck) [reply components/image-combo/image-combo.less]: On the case page `<QuoteBlock>` is a child of `<ImageCombo>`. `<QuoteBlock>` uses this class so we can't delete it.

Comment (vvandijck) [reply components/image-combo/image-combo.less]: I did add a new `hasText` prop and removed `classes`.

File: components/image-combo/image-combo.js [hunk 2]
```diff
@@ -1,22 +1,22 @@
 import PropTypes from 'prop-types';
 
-const ImageCombo = ({ classes = '', children }) => {
+const ImageCombo = ({ hasText = '', children }) => {
```
Comment (Siilwyn) [inline components/image-combo/image-combo.js]: Shouldn't default to a string but a boolean like `false`.

File: components/image-combo/image-combo.less (hunk 1 above)
Comment (Siilwyn) [reply components/image-combo/image-combo.less]: Okay, not sure if this is technical debt or not...

**Pull request merged**